runners/samza/src/main/java/org/apache/beam/runners/samza/translation/GroupByKeyTranslator.java

```diff
@@ -104,7 +104,7 @@ private static <K, InputT, OutputT> void doTranslate(
             outputTag,
             input.isBounded());
 
-    ctx.registerMessageStream(output, outputStream);
+    ctx.registerMessageStream(output, outputStream, false);
```

Comment:
Why we need to pass false here? We need to compute the transform metrics for GBK too, right?

Reply:
GBK will be a different PR, remember the watermark broadcast issue we discussed - we will do calculation for GBK + Shuffle Latency per windowId